### PR TITLE
REPOSITORY: Add `Distributor`

### LINKED ISSUE #42: REPOSITORY: Add `Distributor`

### PR DESCRIPTION
**Pull request by Xurshid-A-J**
Closes #42

### CODE REVIEW

File: CountPad.Infrastructure/Repositories/DistributorRepository.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+namespace CountPad.Infrastructure.Repositories
+{
+    public class DistributorRepository : BaseRepository, IDistributorRepository
+    {
+
+        public async Task<int> AddAsync(Distributor distributor)
+        {
+            using (NpgsqlConnection connection = CreateConnection())
+            {
+
```
Comment (MirzayevDeveloper) [inline CountPad.Infrastructure/Repositories/DistributorRepository.cs:19]: remove extra line

File: CountPad.Application/Interfaces/RepositoryInterfaces/IDistributorRepository.cs [hunk 2]
```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using CountPad.Domain.Models.Distributors;
```
Comment (MirzayevDeveloper) [inline CountPad.Application/Interfaces/RepositoryInterfaces/IDistributorRepository.cs:2]: add  headerfile

File: CountPad.Application/Interfaces/RepositoryInterfaces/IDistributorRepository.cs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+﻿using System;
```
[the remark this answers is not in the document]
Comment (Xurshid-A-J) [reply CountPad.Application/Interfaces/RepositoryInterfaces/IDistributorRepository.cs:1]: Boldi yes qildik

File: CountPad.Infrastructure/Repositories/DistributorRepository.cs [hunk 4]
```diff
@@ -0,0 +1,38 @@
+﻿using System;
```
[the remark this answers is not in the document]
Comment (Xurshid-A-J) [reply CountPad.Infrastructure/Repositories/DistributorRepository.cs:1]: Yes qildik uni ham

File: CountPad.Infrastructure/Repositories/DistributorRepository.cs (hunk 1 above)
Comment (Xurshid-A-J) [reply CountPad.Infrastructure/Repositories/DistributorRepository.cs:19]: Removed

File: CountPad.Application/Services/DistributorService.cs [hunk 5]
```diff
@@ -0,0 +1,22 @@
+﻿using System;
```
[the remark this answers is not in the document]
Comment (Xurshid-A-J) [reply CountPad.Application/Services/DistributorService.cs:1]: done

File: CountPad.Application/Interfaces/RepositoryInterfaces/IDistributorRepository.cs (hunk 2 above)
Comment (Xurshid-A-J) [reply CountPad.Application/Interfaces/RepositoryInterfaces/IDistributorRepository.cs:2]: qushdik

**Pull request merged**